source/jormungandr/jormungandr/scenarios/helper_classes/fallback_durations.py

```diff
@@ -103,9 +103,23 @@ def _get_duration(self, resp, place):
 
     @new_relic.distributedEvent("routing_matrix", "street_network")
     def _get_street_network_routing_matrix(self, origins, destinations):
-        return self._instance.get_street_network_routing_matrix(
-            origins, destinations, self._mode, self._max_duration_to_pt, self._request, **self._speed_switcher
-        )
+        try:
+            return self._instance.get_street_network_routing_matrix(
+                origins,
+                destinations,
+                self._mode,
+                self._max_duration_to_pt,
+                self._request,
+                **self._speed_switcher
+            )
+        except Exception as e:
+            print("Exception':{}".format(str(e)))
+            resp = response_pb2.StreetNetworkRoutingMatrix()
+            row = resp.rows.add()
+            routing = row.routing_response.add()
+            routing.duration = -1
+            routing.routing_status = response_pb2.unknown
```

Comment:
That's strange to return such a non-sense response, why is it needed?

Reply:
So that it doesn't crash afterwards and is treated as an empty response.